app/views/works/index.html.erb

```diff
@@ -0,0 +1,38 @@
+<h2> List of Works </h2>
+
+
+<% [[@albums, "Albums"], [@books, "Books"], [@movies, "Movies"]].each do |work, category| %>
+<h4> <%=category %> </h4>
+
+  <table class="table">
+    <tr>
+      <th> Votes </th>
+      <th> Title </th>
+      <th> Created By </th>
+      <th> Published </th>
+      <th> Upvote </th>
+    </tr>
+  <% work.each do |work| %>
```

Comment:
Notice how these tables are pretty repetitive.  You can DRY them out by using partials.